src/apps/loan-list/list/loan-list.dev.tsx

```diff
@@ -27,6 +27,15 @@ export default {
       defaultValue: 5,
       control: { type: "number" }
     },
+    // Config
+    dangerThresholdConfig: {
+      defaultValue: "0",
+      control: { type: "text" }
+    },
+    warningThresholdConfig: {
```

Comment:
I think the config keys should be more specific. They are globally shared (like the text config keys).

Reply:
any idea for a name? I think they are a little generic, but they should also be the same globally. there will not be separate configurations for them in drupal.

Reply:
Ok. If it is that global then how about making just one configuration config object?
Since it is possible to use a json string we could have one config object called
`patronAlertTresholds`. Does that make sense?
branchesConfig is an example you can look at:
https://github.com/reload/dpl-react/blob/e452336fbb07371befe258ce2cc8fcac81e5577b/src/apps/search-result/search-result.dev.tsx#L93-L98
You can parse the config like this:
https://github.com/reload/dpl-react/blob/e452336fbb07371befe258ce2cc8fcac81e5577b/src/apps/patron-page/util/BranchesDropdown.tsx#L21-L23

Reply:
In one of the components, only one of the two are used